images/release/Dockerfile.centos7

```diff
@@ -0,0 +1,34 @@
+#
+# This is the image that controls the standard build environment for releasing OpenShift Origin.
+# It is responsible for performing a cross platform build of OpenShift.
+#
+# The standard name for this image is openshift/origin-release
+#
+FROM openshift/origin-base
```

Comment:
```
-CMD bsdtar mxzf - && hack/build-cross.sh
+CMD tar mxzf - && hack/build-cross.sh
```
That was one I was going to ask about.  Why are we using bsdtar?
It's not installed by default, and I don't see us using any of it's functionality.  Why not just use tar?